ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -1650,6 +1650,34 @@ def test_tensorflow_zeros_like(  # NOQA
     )
 
 
+# LogSoftmax
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.LogSoftmax",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Please use `helpers.get_dtypes("valid")` here and add a `with_supported_dtypes` decorator to the function. 
Other than that, looks good!